.gitignore

```diff
@@ -8,4 +8,5 @@ hookdocs/
 .idea/*
 *.sql
 *.sqlite
-.DS_Store
\ No newline at end of file
+.DS_Store
+nightingale-companion.zip
```

Comment:
would this be better to be just .zip to follow pattern of above restrictions and be a catch all?